src/NuGet.Services.Validation/Entities/ValidationEntitiesContext.cs

```diff
@@ -762,5 +767,69 @@ private void RegisterSymbolEntities(DbModelBuilder modelBuilder)
                .Property(s => s.RowVersion)
                .IsRowVersion();
         }
+
+        private void RegisterCvsScanEntities(DbModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<ContentScanOperationState>()
+                .HasKey(p => p.Key);
+
+            modelBuilder.Entity<ContentScanOperationState>()
+                .Property(s => s.ValidationStepId)
+                .HasColumnAnnotation(
+                    IndexAnnotation.AnnotationName,
+                    new IndexAnnotation(new[]
+                    {
+                        new IndexAttribute(ContentScanOperationStateValidationStepIdTypeStatusIndex, 0)
+                    }));
+
+            modelBuilder.Entity<ContentScanOperationState>()
+                .Property(s => s.Type)
+                .HasColumnAnnotation(
+                    IndexAnnotation.AnnotationName,
+                    new IndexAnnotation(new[] {
```

Comment:
Please put `{` on a new line:

```suggestion
                    new IndexAnnotation(new[]
                    {
```

Please fix other places as well

Reply:
Updated